distributions/docker/solr/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM solr:8.1.1
+
+COPY maven/replication-solr-managed-schema.xml /opt/solr/server/solr/configsets/_default/conf/managed-schema
+
+ENTRYPOINT  bash -c \
+              precreate-core replication_item; \
+              precreate-core replication_site; \
+              precreate-core replication_status; \
+              precreate-core replication_config; \
+              exec solr -f
+
+EXPOSE 8983
```

Comment:
:pencil2: This is unnecessary, you are actually already inheriting this from the upstream image

Reply:
Removed